### DISCUSSION TITLE
Differences between gltf model display in iOS and Android (AR)

### DISCUSSION THREAD

**Discussion by goout** [google/model-viewer#4190] [Q&A]
### Description

So, for some development purposes, we use scaleX -1 approach on a group that wraps our model (a little difficult hierarchy with animations and ect.) for flipping the model horizontally. And everything works perfectly on Android, but on iOS it seems like faces are flipped in AR. But viewing in Safari is ok.

![photo_2023-03-31_15-40-18](https://user-images.githubusercontent.com/2082785/229122764-4e37e521-b489-46eb-ab99-50de371efc81.jpg)

![photo_2023-03-31_15-33-03](https://user-images.githubusercontent.com/2082785/229122781-78563cdd-b6f6-4716-80aa-d483d50719a9.jpg)

#### Live Demo
[charming-toothpaste.surge.sh](https://charming-toothpaste.surge.sh/)

### Version

- model-viewer: v1.12.0

### OS

- [x] iOS 16.3

### AR

- [x] WebXR
- [x] QuickLook

**Comment by elalish** [comment]
Okay, so the difference is iOS AR only supports USDZ, while all other rendering uses the GLB, so this is a problem of our auto-conversion to USDZ via the three.js USDZExporter (FYI @mrdoob). Meanwhile, you can work around any USDZ auto-conversion problem by doing the conversion yourself with Apple's tools and setting it as our `ios-src`.

**Reply by goout** [reply]
ok, thanks

**Reply by mrdoob** [reply]
As far as I remember, the issue here was Apple's renderer not supporting `scale.x = -1`.

**Reply by elalish** [reply]
FYI @grorg regarding USDZ interop issue.
